gateway/ln-gateway/src/lightning/ldk.rs

```diff
@@ -269,8 +269,6 @@ impl ILnRpcClient for GatewayLdkClient {
         &self,
         _num_route_hints: usize,
     ) -> Result<GetRouteHintsResponse, LightningRpcError> {
-        // TODO: Return real route hints. Not strictly necessary but would be nice to
-        // have.
```

Comment:
Why remove? I think the LDK use-case will lend itself to adding route-hints: if your node only has one channel to an LSP you don't want to bother everyone by announcing it, you'd rather include it as a route hint.

Reply:
`ILnRpcClient::routehints()` is only ever used for LNv1 payment receives. Specifically, the route hints are inserted into the gateway announcement, which can be used by clients when they generate invoices. The LDK gateway will never support LNv1 payment receives, it only supports LNv1 sends. LNv2 payment receives require that the gateway's lightning node generates invoices rather than the fedimint client, so it is able to insert the proper route hints on its own.

Reply:
Right, wasn't thinking about that!

Reply:
I added this context to `GatewayLdkClient::routehints()` for future readers